src/ui/features/projects/ProjectForm/Banner.tsx

```diff
@@ -2,12 +2,37 @@ import React from 'react';
 import { useFormState } from 'react-final-form';
 import { PageBanner } from '@gpn-prototypes/vega-ui';
 
-export const Banner: React.FC = () => {
-  const state = useFormState();
+import { ReferenceDataType } from '../../../../pages/project/types';
+
+type BannerProps = {
+  referenceData: ReferenceDataType;
+};
+
+const getDescription = (
+  region: string | undefined,
```

Comment:
```suggestion
  regionVid: string | undefined,
```
кажется более читаемым

Reply:
Согласен. Исправил